### ISSUE TITLE
Always loading react-hot-loader production AppContainer

### ISSUE THREAD

**Issue by boba-keyost** [milankinen/livereactload#159]
If loader ignores changes in node_modules packages then it uses default require function to load 'react-hot-loader' module. And this module doesn't have 'module.hot' property so it always loads production version.

**Comment by milankinen** [comment]
Hello! Sorry but I couldn't get the problem yet. Could you rephrase the issue?

**Comment by boba-keyost** [comment]
Hm, ok.
If you're using multiple bundles with reloadable application and non reloadable vendor libs(e.g. react etc.) you should add react-hot-loader package to reloadable bundle.

**Comment by milankinen** [comment]
Correct, vendor libs are not hot-reloaded because the assumption is that developer does not modify their content. Of course new libraries can be added and existing ones can be updated but in that case, you need to restart Browserify. And there is no multi-bundle support yet, see #146.
